extensions/machine-learning/src/views/models/prediction/columnsTable.ts

```diff
@@ -205,8 +204,9 @@ export class ColumnsTable extends ModelViewBase implements IDataComponent<Predic
 				width: this.componentMaxLength + 20,
 				justifyContent: 'flex-start'
 			}).component();
-			const warningButton = this.createWarningButton(constants.outputColumnDataTypeNotSupportedWarning);
+			const warningButton = this.createWarningButton();
 			warningButton.onDidClick(() => {
```

Comment:
please use constant.mlExtDocLink link for now. we don't have doc for this so we can use the general one